chore: update production manifest with new tmp dir

crozzy opened the pull request:
Since https://github.com/quay/clair/pull/2019 clair will now use /var/tmp instead of /tmp by default to store the manifest layers. This PR updates where the PVC is mounted in order to concide with the new default.

crozzy committed:
chore: update production manifest with new tmp dir

Since https://github.com/quay/clair/pull/2019 clair will now use
/var/tmp instead of /tmp by default to store the manifest layers. This
PR updates where the PVC is mounted in order to concide with the new
default.

Signed-off-by: crozzy <joseph.crosland@gmail.com>

hdonnay commented on contrib/openshift/manifests/manifests.yaml:
```diff
@@ -56,7 +56,7 @@ objects:
                 - sh
                 - '-c'
-                - rm -rf /tmp/sha* /tmp/fetch.* /tmp/rpmscanner.* && exec clair
+                - rm -rf /var/tmp/sha* /var/tmp/fetch.* /var/tmp/rpmscanner.* && exec clair
```
The `rm`s shouldn't be needed, now.

Is there some way to double-check that?

crozzy replied:
We can remove them and check the PVC metrics once it's deployed

crozzy committed:
contrib: remove rms that were needed for previous fetcher

Since https://github.com/quay/claircore/pull/1061 we should no longer
need the pre-removal of fetch layers.

Signed-off-by: crozzy <joseph.crosland@gmail.com>

hdonnay reviewed (approved):
LGTM w/rebase

crozzy commented:
/fast-forward
